application/controllers/app.rb

```diff
@@ -564,13 +564,17 @@ class App < Roda
               scores.each_key do |key|
                 title[key] = case key
```

Comment:
Let's discuss this switch statement at our next meeting (please put on Trello!) I'm curious what else `title[key]` is used for other than this switch statemeent.

Reply:
No problem, I'll put it on the Moonbear Trello board!
And should I merge this branch to the master right now or after our meeting on Friday?

Reply:
Merge into develop for now and you can confirm it after our Friday meeting?

Reply:
OK, done!